<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightingZone</class>
 <widget class="QWidget" name="LightingZone">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="ZoneName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 15px;
	font-weight: 500;
}
</string>
   </property>
   <property name="text">
    <string>ZoneName</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ModeDropdown">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>140</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>60</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Brightness:</string>
   </property>
  </widget>
  <widget class="QSlider" name="BrightnessSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>9</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="ColorButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
  <widget class="QSlider" name="SpeedSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>205</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>9</number>
   </property>
   <property name="maximum">
    <number>9</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="RandomCheckbox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>16</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="DirectionDropdown">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Clockwise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Counterclockwise</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Random:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
